### PR TITLE
Restore Browse our FAQ

### PR DESCRIPTION
**Pull request by ravishanker**
Restore Browse our FAQ row on help screen for WP

To test:

- Navigate to Me > App Settings > Debug settings
- Enable **wordpress_support_forum_remote_field** under Remote features and Restart the app (scroll down if required)
- Navigate to Me > Help & Support
- Verify that the screen is as shown above
- Tap on the **Community forums** row
- Verify that it opens the new Mobile Forum on Wordpress.org in the browser
- Tap on the **Browse our FAQ** row
- Verify that it open Mobile App Support page on the web

## Regression Notes
1. Potential unintended areas of impact
None

2. What I did to test those areas of impact (or what existing automated tests I relied on)
Manual testing

3. What automated tests I added (or what prevented me from doing so)
Existing unit tests

PR submission checklist:

- [x] I have completed the Regression Notes.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

**Comment by staskus** [comment]
To confirm, it will be visible on both Jetpack and WordPress, correct?

**Comment by irfano** [comment]
> To confirm, it will be visible on both Jetpack and WordPress, correct?

I think we should wait for a response to the question I asked in the design file before merging this PR and adding it to iOS.
Internal ref: qk5s4q4mZsXknwhMWo2vIY-fi-4769%3A14988

**Comment by staskus** [comment]
> I think we should wait for a response to the question I asked in the design file before merging this PR and adding it to iOS.

Completely agree 👍

**Comment by guarani** [comment]
> I think we should wait for a response to the question I asked in the design file before merging this PR and adding it to iOS.
> Internal ref: [qk5s4q4mZsXknwhMWo2vIY-fi-4769%3A14988](https://href.li/?https://www.figma.com/file/qk5s4q4mZsXknwhMWo2vIY/?node-id=4769%3A14988)

👋 @irfano, as discussed in internal ref p1676551924520159/1675873769.529629-slack-C3EA6SMFH, let's put this button back in.

**Review by irfano — APPROVED** [review]
Minor ( 🔍️ ): FAQ button could be above the divider as how it was before. It's not crucial, and we may update it in a **separate PR**.

LGTM! Approving. 👍🏻

**Pull request merged**
